spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/FilteringBatchMessageListenerAdapter.java

```diff
@@ -71,22 +72,24 @@ public FilteringBatchMessageListenerAdapter(BatchMessageListener<K, V> delegate,
 
 		super(delegate, recordFilterStrategy);
 		this.ackDiscarded = ackDiscarded;
+		this.consumerAware = this.delegateType.equals(ListenerType.ACKNOWLEDGING_CONSUMER_AWARE) || this.delegateType.equals(ListenerType.CONSUMER_AWARE);
```

Comment:
The code line  length `! > 120` symbols.

Reply:
Thanks for your comment 🙇‍♂️ 
I checked all the code and fixed it! 

I think it would be better to include this rule to `checkStyle`. I will make new PR for this.